examples/preprocessing/plot_movement_compensation.py

```diff
@@ -44,15 +54,18 @@
 
 topo_kwargs = dict(times=[0, 0.1, 0.2], ch_type='mag', vmin=-500, vmax=500)
 
-# 0. Take average of stationary data (bilateral auditory patterns)
+###############################################################################
+# First, take the average of stationary data (bilateral auditory patterns).
 evoked_stat = mne.Epochs(raw_stat, events, 1, -0.2, 0.8).average()
 evoked_stat.plot_topomap(title='Stationary', **topo_kwargs)
 
-# 1. Take a naive average (smears activity)
+###############################################################################
+# Second, tak a naive average (smears activity).
```

Comment:
take

Reply:
apart from 'take' I'd be a little more verbose in this point (mentioning that the average is taken from the dataset with simulated movement, without movement compensation which results in smeared activity).